exercises/saddle-points/.meta/template.j2

```diff
@@ -0,0 +1,46 @@
+"""Tests for the saddle-points exercise
+
+Implementation note:
+The saddle_points function must validate the input matrix and raise a
+ValueError with a meaningful error message if the matrix turns out to be
+irregular.
+"""
+{%- import "generator_macros.j2" as macros with context -%}
+
+{% macro test_case(case) -%}
+    {%- set input = case["input"] -%}
+    {%- set expected = case["expected"] -%}
+    def test_{{ case["description"] | to_snake }}(self):
+        {% if input["matrix"][0] -%}
+        matrix = {{ input["matrix"] }}
+        {% else -%}
+        matrix = []
+        {% endif -%}
+        {% if case is error_case -%}
+        with self.assertRaisesWithMessage(ValueError):
+            {{- case["property"] | to_snake }}(matrix)
+        {% else -%}
+        self.assertEqual(
+        sorted({{ case["property"] | to_snake }}(matrix), key=lambda d: (d["row"],d["column"])),
+        sorted({{ expected }}, key=lambda d: (d["row"],d["column"]))
+        )
+        {% endif -%}
+{% endmacro %}
```

Comment:
```suggestion
{% endmacro -%}
```

To remove the added blank line before `import unittest`

Reply:
Done. Had to also, remove the white space eliminator from the beginning of the import macros call so that there still be a single newline between the comment and the first line of code.